tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "composite": true,
     "sourceMap": true,
     "allowJs": true,
+    "skipLibCheck": true,
```

Comment:
not that i disagree with this setting, but did something break in this pr which forced you to turn this on?